shell/src/main/java/tachyon/shell/command/CommandUtils.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.shell.command;
+
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+
+import tachyon.Constants;
+import tachyon.TachyonURI;
+import tachyon.client.file.TachyonFile;
+import tachyon.client.file.TachyonFileSystem;
+import tachyon.client.file.options.SetStateOptions;
+import tachyon.client.lineage.TachyonLineage;
+import tachyon.exception.TachyonException;
+import tachyon.thrift.FileInfo;
+import tachyon.thrift.LineageInfo;
+import tachyon.util.FormatUtils;
+
+/**
+ * Common util methods for executing commands.
+ */
+public class CommandUtils {
+
+  private CommandUtils() {
+    // Not intended for instantiation.
+  }
+
+  /**
+   * Set a new TTL value or unset an existing TTL value for file at path.
+   *
+   * @param path the file path
+   * @param ttlMs the TTL (time to live) value to use; it identifies duration (in milliseconds) the
+   *        created file should be kept around before it is automatically deleted, irrespective of
+   *        whether the file is pinned; {@link Constants#NO_TTL} means to unset the TTL value
+   * @throws IOException when failing to set/unset the TTL
+   */
+  public static void setTTL(TachyonFileSystem tfs, TachyonURI path, long ttlMs) throws IOException {
```

Comment:
It's currently used by both `setTTL` and `unsetTTL`. They are different enough that it isn't worth making a common parent for them